src/main/java/com/google/sps/servlets/GameQuizStatusServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.servlets;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.sps.QuizTimingPropertiesUtils;
+import com.google.sps.UserUtils;
+ 
+//This servlet gets the quiz question from the Game entity 
+//The quiz question is updated if the quiz timestamp is outdated
+@WebServlet("/game-quiz-status-servlet")
+public class GameQuizStatusServlet extends HttpServlet {
+ 
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+ 
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Cookie cookies[] = request.getCookies();
+        if(cookies == null){
+            response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+            return;
+        }
+        Entity userEntity = UserUtils.getUserFromCookie(cookies, datastore);
+        if(userEntity == null){
+            response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+            return;
+        }
+ 
+        QuizTimingPropertiesUtils timing_properties = new QuizTimingPropertiesUtils();
+        UserUtils user_utils_class = new UserUtils();
```

Comment:
u could just make this a static function so you wouldn't need a class instance

Reply:
you mean timing_properties? It is actually the whole class that I am instantiating here

Reply:
Instead of instantiating these classes, make the methods you need static. Then you can call `UserUtils.getEntityFromDatastore("Game", "currentGame", etc, etc)` without making an instance of `UserUtils`. And ditto for `QuizTimingPropertyUtils`.

Reply:
Did this!